## Intervalo de confianza de la media con varianza conocida

### Ejercicio 1

La empresa AUTOZIN produce partes de componentes cilíndricos para la industria automotriz que tienen un promedio de diámetro de 5 milímetros con una distribución normal y una desviación estándar de 0.1 milímetros. Para confirmar esto se tomó una muestra de 25 partes seleccionadas al azar que muestra que el diámetro promedio es de 5.027 milímetros. Encuentre el intervalo de confianza del 95% para la media de la población de todas las partes de componentes cilíndricos que produce la empresa.

**Se empleara:**
$$
IC{:}\quad\bar{X}-Z_{(1-\frac\alpha2)}\cdot\frac\sigma{\sqrt{n}}\leq\mu\leq\bar{X}+Z_{(1-\frac\alpha2)}\cdot\frac\sigma{\sqrt{n}}
$$

Los parámetros relevantes son:
- Desviacion estandar ( $\sigma$ ) = 0.1
- Promedio ($\bar{X}$) = 5.027
- Tamaño de muestra ($n$) = 25
- Confianza = 95 %

**Si:**
1 - $\alpha$ = 0.95

**Entonces:**
- $\alpha$ = 0.05

In [4]:
import scipy.stats as st
import math
# Asignando valores:
O = 0.1
X = 5.027
n = 25
# a -> calculado previamente
a = 0.05

# Calculamos Z
Z = round((st.norm.ppf(1-(a/2))),2)

# Reemplazamos en la formula IC
IC1 = X-(Z*O/math.sqrt(n)) 
IC2 = X+(Z*O/math.sqrt(n)) 

print(Z)
print("IC:",IC1," u ",IC2)

1.96
IC: 4.9878  u  5.0662


Con un nivel de confianza del 95%, el diámetro promedio de los componentes cilíndricos está comprendido desde 4.9878 a 5.0662 mil.